curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ec19d5268da7074941f84.md

```diff
@@ -6,8 +6,17 @@ dashedName: step-32
 ---
 
 # --description--
+In order to query two media conditions, the `and` logical operator is used. This one of many [logical operators](https://www.freecodecamp.org/news/media-query-css-example-max-and-min-screen-width-for-mobile-responsive-design/) used to construct media queries.
```

Comment:
The challenge should not contain outbound links.

> [Challenge descriptions/instructions](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions)
> 
> Don't use outbound links. These interrupt the flow. Campers should never have to google anything during these challenges.

Reply:
Ill remove the link and convert to PR.